modules/optionals/cloud9/app.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+import aws_cdk
+from aws_cdk import App, Aws, CfnOutput
+
+from stack import Cloud9Stack
+
+deployment_name = os.getenv("ADDF_DEPLOYMENT_NAME")
+module_name = os.getenv("ADDF_MODULE_NAME")
+
+def _param(name: str) -> str:
+    return f"ADDF_PARAMETER_{name}"
+
+image_id = os.getenv(_param("IMAGE_ID"), "ami-0beaa649c482330f7")
```

Comment:
I assume this image is region-specific....I think you need a map based on region code?

Reply:
Agreed. See my comment in the README.md